### PR TITLE
Typo, unused reference, and a couple other small corrections

### PR DESCRIPTION
**Pull request by zgrummons**
## Description
This PR makes five total corrections:
1. Connectionstring -> ConnectionString 
The console message when you fail to specify either ConnectionString or QueueName is spelled "-Connectionstring". The argument parser is case-sensitive, so if you copy-paste the flag from the console window, the program will not run. 
2. Removed unused reference at line 20:
var p = new Program();
3. Changed the arguments parsing to use a switch block
4. Changed empty string references to their corresponding string methods. This includes initialization of strings to string.Empty and checking for empty strings to use string.IsNullOrEmpty().
5. Linting. Used var where possible, changed variable names to be context-appropriate.

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
- [x] Title of the pull request is clear and informative.
- [x] If applicable, the code is properly documented.
- [x] The code builds without any errors.

### CODE REVIEW

File: samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs [hunk 1]
```diff
@@ -8,41 +8,41 @@ namespace BasicSendReceiveQuickStart
...
 
         static void Main(string[] args)
         {
-            string ServiceBusConnectionString = "";
-            string QueueName = "";
+            string serviceBusConnectionString = string.Empty;
+            string queueName = string.Empty;
 
-            for (int i = 0; i < args.Length; i++)
+            for (var i = 0; i < args.Length; i++)
```
Comment (SeanFeldman) [inline samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs]: I wonder why a `var` here, but not two lines above?

File: samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs [hunk 2]
```diff
@@ -8,41 +8,41 @@ namespace BasicSendReceiveQuickStart
 {
     class Program
     {
-        static IQueueClient queueClient;
+        static IQueueClient _queueClient;
```
Comment (SeanFeldman) [inline samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs]: Prefixing with an underscore is not a standard practice. The repo doesn't have it as a standard either.
I would suggest to remove it.

File: samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs [hunk 3]
```diff
@@ -8,41 +8,41 @@ namespace BasicSendReceiveQuickStart
...
-                if (args[i] == "-ConnectionString")
+                switch (args[i])
                 {
-                    Console.WriteLine($"ConnectionString: {args[i+1]}");
-                    ServiceBusConnectionString = args[i + 1]; // Alternatively enter your connection string here.
+                    case "-ConnectionString":
+                        Console.WriteLine($"ConnectionString: {args[i+1]}");
+                        serviceBusConnectionString = args[i + 1]; // Alternatively enter your connection string here.
+                        break;
+                    case "-QueueName":
```
Comment (SeanFeldman) [inline samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs]: 👍

Comment (zgrummons) [reply samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs]: I thought about going one step further and using a new 8.0 switch expression here, but didn't want to bewilder newer devs

File: samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs (hunk 1 above)
Comment (zgrummons) [reply samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs]: Good point, changed.

File: samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs (hunk 2 above)
Comment (zgrummons) [reply samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs]: I see, the underscore is just what I've always done for member variables. Reverted.

**Pull request merged**